src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -54,6 +54,9 @@ export class CalciteCombobox {
 
   @Prop() placeholder?: string;
 
+  /** specify the maximum number lines to display before showing the scroller */
```

Comment:
maybe just say maximum number of combobox-items? Items could potentially wrap their text to multiple lines so maybe this is a bit confusing based on [the question we got about max-items in dropdown](https://github.com/Esri/calcite-components/pull/1343)?

Reply:
Yeah I wasn't totally sure how to word it. It's a bit different because a combobox item can have multiple levels of combobox items inside it. So it's like "how many items including children" to display or something?